apps/OpenSpace/main.cpp

```diff
@@ -997,13 +997,31 @@ std::string selectedSgctProfileFromLauncher(LauncherWindow& lw, bool hasCliSGCTC
             }
         }
         else {
-            if (std::filesystem::path(config).extension() == ".xml") {
-                //user customzied sgct config
+            std::filesystem::path c = absPath(config);
+            
+            std::filesystem::path cj = c;
+            cj.replace_extension(".json");
+            
+            std::filesystem::path cx = c;
+            cx.replace_extension(".xml");
+
+            if (c.extension().empty()) {
+                if (std::filesystem::exists(cj)) {
```

Comment:
This makes it impossible to load an XML file with the same name if a JSON version with that name already exists. The JSON version will always be the one that is loaded

Very confusing if you have used the conversion script to convert an old XML file to JSON, and want to try each of them individually

Reply:
This could will only take care of people selecting the configuration from the launcher.  If you put `SGCTConfig = "${CONFIG}/foobar.xml` that XML file will be loaded instead.  And I think we only want to support the JSON files from the Launcher?

Reply:
You're probably right. But then the XML files shouldn't show up in the launcher at all :)

Reply:
:+1: I would agree;  we decided to only deprecate the support for XML formats and not remove it for the next release, so it would be weird for them to still be supported but not show up.
Maybe the solution is to add the extension to the configuration files shown in the launcher?

Reply:
Ah, right. Yeah, it makes sense to show the extension in the launcher, and fix it so that any of the files can be read. 

Another idea is to update the visual appearance of the XML files in the launcher so that it "warns" the user that these will soon be deprecated. That way we would actually target the people who use the file type and inform them how to convert it to the new json format :) Should be relatively easy

Reply:
I did that in this PR now.
1. The extensions are shown in the launcher
2. All XML files are in dark yellow color 
3. If at least one XML file is in the list, the dropdown menu gets a tooltip informing that they need to convert their files by the next version

Reply:
Great :) Tried it out quickly and did not see the tooltip though